build.gradle

```
@@ -105,4 +108,34 @@ subprojects {
     dependencies {
         testImplementation 'ch.qos.logback:logback-classic:1.2.3'
     }
+
+    spotless {
+        enforceCheck false
+        java {
+            removeUnusedImports()
+            trimTrailingWhitespace()
+            endWithNewline()
+
+            prettier(['prettier': '2.5.1', 'prettier-plugin-java': '1.6.1'])
+                .config([
+                    'parser'    : 'java',
+                    'tabWidth'  : 4,
+                    'printWidth': 120
+                ])
+
+            importOrder('', 'java', 'javax', '\\#')
+        }
+        groovyGradle {
+            target '**/*.groovy'
+            greclipse('4.19.0')
+        }
+    }
+
+    checkstyle {
+        toolVersion = "9.3"
+        configFile = rootProject.file('config/checkstyle/checkstyle.xml')
+        checkstyleTest.enabled = false
+        checkstyleMain.enabled = false
```

We probably don't need these now that the failures are ignored?